src/phoenix/server/api/types/Span.py

```diff
@@ -15,19 +17,18 @@
 from phoenix.server.api.types.Evaluation import DocumentEvaluation, SpanEvaluation
 from phoenix.server.api.types.MimeType import MimeType
 from phoenix.trace.schemas import ComputedAttributes, SpanID
-from phoenix.trace.semantic_conventions import (
-    EMBEDDING_EMBEDDINGS,
-    EMBEDDING_VECTOR,
-    EXCEPTION_MESSAGE,
-    INPUT_MIME_TYPE,
-    INPUT_VALUE,
-    LLM_TOKEN_COUNT_COMPLETION,
-    LLM_TOKEN_COUNT_PROMPT,
-    LLM_TOKEN_COUNT_TOTAL,
-    OUTPUT_MIME_TYPE,
-    OUTPUT_VALUE,
-    RETRIEVAL_DOCUMENTS,
-)
+
+EMBEDDING_EMBEDDINGS = SpanAttributes.EMBEDDING_EMBEDDINGS
+EMBEDDING_VECTOR = EmbeddingAttributes.EMBEDDING_VECTOR
+EXCEPTION_MESSAGE = OTELSpanAttributes.EXCEPTION_MESSAGE
```

Comment:
thought: If this is all we need opentelemetry semcov for, it's probably not worth the separate package.

Reply:
good point. it has been removed